templating/hinclude.rst

```diff
@@ -1,10 +1,10 @@
 .. index::
     single: Templating; hinclude.js
 
-How to Embed Asynchronous Content with hinclude.js
+How to Embed Asynchronous Content with HInclude.js
 ==================================================
 
-Controllers can be embedded asynchronously using the `hinclude.js`_ JavaScript library.
+Controllers can be embedded asynchronously using the `HInclude.js`_ JavaScript library.
```

Comment:
Super minor but ... according to http://mnot.github.io/hinclude/, this is called `HInclude` and `hinclude.js` is only used to refer to the specific JS file you need to load in your pages.

Reply:
Wow, great find. You're completely correct, I've fixed it (also in the http cache and reference articles)